tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -18,34 +18,43 @@ class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
     """
+
     def test_add_node(self):
         """
         Test for adding worker nodes to the cluster while IOs
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
             before_replica_counts = dict()
-            count = 2
             machines = machine_utils.get_machinesets()
             for machine in machines:
-                before_replica_counts.update({machine: machine_utils.get_replica_count(machine)})
-            worker_nodes_before = helpers.get_worker_nodes()
-            logger.info(f'The worker nodes number before adding a new node is {len(worker_nodes_before)}')
+                before_replica_counts.update(
+                    {machine: machine_utils.get_replica_count(machine)}
+                )
+                logger.info(machine_utils.get_replica_count(machine))
+            logger.info(f'The worker nodes number before {len(helpers.get_worker_nodes())}')
             after_replica_counts = dict()
+            total_count = 0
             for machine in machines:
-                machine_utils.add_node(machine, count=count)
-                after_replica_counts.update(({machine: machine_utils.get_replica_count(machine)}))
+                machine_utils.add_node(
+                    machine, count=machine_utils.get_replica_count(machine) + 1
+                )
+                after_replica_counts.update(
+                    ({machine: machine_utils.get_replica_count(machine)})
+                )
+                total_count += machine_utils.get_replica_count(machine)
+                logger.info(total_count)
```

Comment:
IF it's 3 az you can do add_node once in anyone of the az as per your code you are increasing from all az